sbin/spark-daemon.sh

```diff
@@ -23,6 +23,7 @@
 #
 #   SPARK_CONF_DIR  Alternate conf dir. Default is ${SPARK_HOME}/conf.
 #   SPARK_LOG_DIR   Where log files are stored. ${SPARK_HOME}/logs by default.
+#   SPARK_LOG_MAX_FILES Max log files of Spark daemons can rotate to, at least 5.
```

Comment:
I think you can just make it default 5. If it's lower then 5, it's okay because it's what the user set.